bindings/java/src/integration/com/apple/foundationdb/MappedRangeQueryIntegrationTest.java

```diff
@@ -194,8 +194,9 @@ public interface RangeQueryWithIndex {
 		try {
 			List<MappedKeyValue> kvs =
 			    tr.getMappedRange(KeySelector.firstGreaterOrEqual(indexEntryKey(begin)),
-			                          KeySelector.firstGreaterOrEqual(indexEntryKey(end)), MAPPER,
-			                          ReadTransaction.ROW_LIMIT_UNLIMITED, false, StreamingMode.WANT_ALL)
+			                      KeySelector.firstGreaterOrEqual(indexEntryKey(end)), MAPPER,
+			                      ReadTransaction.ROW_LIMIT_UNLIMITED, 0, false,
```

Comment:
0 means `MATCH_INDEX_ALL`? if so, please define a const to be more readable.

Reply:
yeah, sounds good